### DISCUSSION TITLE
Project page serving from the wrong URL [SOLVED]

### DISCUSSION THREAD

**Discussion by ZhengRaymond** [community/community#21840] [Other Feature Feedback, Questions, & Ideas]
<p>When I go to my custom domain <a href="http://example.com" rel="nofollow noopener">http://example.com</a>, nothing appears. The network requests are attempting to fetch the static files from <a href="http://example.com/PROJECT_NAME/static/main.js" rel="nofollow noopener">http://example.com/PROJECT_NAME/static/main.js</a>, when they are actually located at <a href="http://example.com/static/main.js." rel="nofollow noopener">http://example.com/static/main.js. </a></p>

**Comment by ZhengRaymond** [comment, marked as answer]
<p>Solution:</p>
<p>Setting package.json’s “homepage” field to “<a href="http://name.github.io" rel="nofollow noopener">name.github.io</a>” instead of “<a href="http://name.github.io/project_name" class="inline-onebox-loading" rel="nofollow noopener">name.github.io/project_name</a>”.</p>
<p>Since DNS providers route to <a href="http://name.github.io" rel="nofollow noopener">name.github.io</a> for all (even multiple) projects, and uses CNAME to identify the project, the static files should be served from <a href="http://name.github.io" rel="nofollow noopener">name.github.io</a>, not <a href="http://name.github.io/project_name" class="inline-onebox-loading" rel="nofollow noopener">name.github.io/project_name</a>. </p>

**Comment by sarojkumar007** [comment]
<p>I need an explanation of this. What to edit in the package.json file.</p>
